### PR TITLE
Mention the need to define CROSS_COMPILE_ARM32 in newer kernels

### PR DESCRIPTION
**Pull request by pablomh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,20 +34,22 @@ NOTE: I am not going to write this for beginnings. I assume if you are smart eno
 ```bash
 make O=out ARCH=arm64 <defconfig>
 
-PATH="<path to clang folder>/bin:<path to gcc folder>/bin:${PATH}" \
+PATH="<path to clang folder>/bin:<path to gcc folder>/bin:<path to 32 bit gcc folder>/bin:${PATH}" \
```
Comment (nathanchance) [inline README.md]: Mind clarifying `64-bit gcc folder` then?

Comment (pablomh) [reply README.md]: Do you mean: `s/<path to gcc folder>/<path to 64-bit gcc folder>/` ???

Comment (nathanchance) [reply README.md]: Yes!

**Comment by nathanchance** [comment]
Thanks!

**Pull request merged**